spec/factories/api/definition.rb

```diff
@@ -0,0 +1,19 @@
+FactoryBot.define do
+  factory :definition, class: 'Api::Definition' do
```

Comment:
Is it worth namespacing all the factories for consistency - ie, `api_definition` ? Would avoid potential future name classes with models as well?

Reply:
I reckon it might. Let's see if we collide in future and I'll migrate them then.